nixos/modules/services/web-apps/wordpress.nix

```diff
@@ -305,6 +305,9 @@ in
       { assertion = cfg.database.createLocally -> cfg.database.user == user;
         message = ''services.wordpress.sites."${hostName}".database.user must be ${user} if the database is to be automatically provisioned'';
       }
+      { assertion = db.createLocally -> db.passwordFile == null;
```

Comment:
would need to look at the code but wouldn't it be possible to support that? I don't know if it's worth it though

Reply:
Would be cool, but someone who needs it should implement it in an other PR. The check implemented here is also present in the module bookstack and soon invoiceplane, so we would have some kind of feature parity.